openmsupply/open-msupply
# 1222 programs hiv care & treatment's encounter part 02

Fixes #1222

# 👩🏻‍💻 What does this PR do? 
- Viral load status based on VL result
- VL trend chart shows 1000 threshold
- Enable location for type of visit only if visit is refill or outreach

# 🧪 How has/should this change been tested? 
- Go to VR load and enter a number in result to see status change
- Go to General -> enter visit type -> see location enabled
